src/white-bread.md

```diff
@@ -0,0 +1,38 @@
+# White Sandwich Bread
+
+![White Bread](pix/white-bread.webp)
+
+Simple white bread. Like the bread you get from the grocery store but 100x better.
+
+- ⏲️ Prep time:     10 min
+- ⬆️ Proofing time: 1.5 - 2.5 hrs
+- 🍞Baking time:   35 min - 1 hr
+
+## Ingredients
+
+- 3/4 cup (174g) water
+- 1/2 cup (125g) whole milk
+- 1 packet (9g) active dry yeast (or 6.75g - you can round up to 7g if your scale doesn't have decimals -  instant yeast)
+- 3 cups (440g) bread flour (or 408g all purpose flour + 32g vital wheat gluten)
+- 1 ½ tsp (8g) fine sea salt
+- 1 tbsp (21g) granulated sugar
+- 3 tbsp (42g) unsalted butter, softened (you can substitute with 31.5g of flavorless oil like sunflower oil, vegetable oil, coconut oil, light olive oil etc.)
```

Comment:
perhaps specify to use vegetable oil? or not.. I wonder what bacon grease would taste like in white bread...